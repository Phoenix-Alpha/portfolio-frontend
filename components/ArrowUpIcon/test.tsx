import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { ArrowUpIcon } from '.'

describe('<ArrowUpIcon />', () => {
  it('should renders correct <ArrowUpIcon />', () => {
    const wrapper = shallow(<ArrowUpIcon />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
