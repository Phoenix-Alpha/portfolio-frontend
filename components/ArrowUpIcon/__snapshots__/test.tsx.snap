// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ArrowUpIcon /> should renders correct <ArrowUpIcon /> 1`] = `
<svg
  className="arrow-up-icon"
  fill="none"
  height={6}
  viewBox="0 0 12 6"
  width={12}
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M1.5 5.25L6 0.75L10.5 5.25"
    strokeLinecap="round"
    strokeLinejoin="round"
    strokeWidth="1.5"
  />
</svg>
`;
